cmds.sh

```
@@ -0,0 +1,7 @@
+python -m ehr2vec.main.01_create_data
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)